views/layouts/partials/header.tpl.php

```diff
@@ -1 +1,8 @@
-<div>Header</div>
\ No newline at end of file
+<div>
+    <div>
```

Comment:
Menu should be an ```<ul>``` instead of ```<div>```, and each item should be included in a ```<li>```
so... you would have:
```
<ul>
<li><a href="/">Home</a></li>
....
</ul>
```